From 8122aa9a8fa8920cc17dad4540e9f74cbe996488 Mon Sep 17 00:00:00 2001
From: "Arnout Vandecappelle (Essensium/Mind)" <arnout@mind.be>
Date: Wed, 20 Jan 2021 14:21:17 +0100
Subject: [PATCH] mac80211.sh: don't disable auto_channel on repeaters

Since UGW 8.4.2, auto_channel (automatic channel configuration) is
disabled on repeaters (i.e. if there is a STA interface configured).

Currently prplMesh requires ACS to complete radio slave join, so until 
PPM-1076 is fixed disable auto_channel.

Signed-off-by: Alex Kanter <alex.kanter@devalore.com>
Signed-off-by: Arnout Vandecappelle (Essensium/Mind) <arnout@mind.be>
---
diff --git a/files/scripts/lib/netifd/wireless/mac80211.sh b/files/scripts/lib/netifd/wireless/mac80211.sh
index 809388a..754ca9f 100755
--- a/files/scripts/lib/netifd/wireless/mac80211.sh
+++ b/files/scripts/lib/netifd/wireless/mac80211.sh
@@ -1354,7 +1354,8 @@ drv_mac80211_setup_phy() {
 }
 
 check_repeater_mode() {
-	auto_channel=0
+	//disable auto channel untill prplmesh PPM-1076 is fixed 
+	//auto_channel=0  
 	channel=first_non_dfs
 	htmode="auto"

